lib/jsdom/living/post-message.js

```diff
@@ -21,11 +21,11 @@ module.exports = function (message, targetOrigin) {
   }
 
   // TODO: event.source - requires reference to source window
-  // TODO: event.origin - requires reference to source window
   // TODO: event.ports
   // TODO: event.data - structured clone message - requires cloning DOM nodes
   const event = new this.MessageEvent("message", {
-    data: message
+    data: message,
+    origin: targetOrigin
```

Comment:
I have reverted the commit. Any idea how to pass in the origin when we register the event handler? I guess I'll just open up an issue and let someone more experienced handle it.